src/scss/components/carousel.scss

```diff
@@ -85,11 +85,11 @@
 }
 
 // Carousel content
-.carousel-content > p {
+.carousel-content p {
   // Mimic .lead
   // Increase distance between text and actions
   margin-bottom: $line-height-computed * 1.25;
-  font-size: floor(($font-size-base * 1.15));
+  font-size: $font-size-lg;
```

Comment:
I don't think font-size-lg is big enough - or at least it doesn't match the old site. I would suggest using:
`$font-size-base * 1.5 `
...which comes out as 2.4rem.
As a side note, and something which occurred to me during updating the watea theme, is that the floor function is not needed now that we are using rems. It works with pixels where that calculation inside the floor function might be equal to 24.4444 and the floor function would have modified this to 24px - which makes sense. However with rems if the calc inside floor comes up with 2.444, we would want to use that value for our rems and not 2 (which would be approx 4px smaller).

Reply:
Nice, done!